applications/asset_tracker/CMakeLists.txt

```diff
@@ -8,7 +8,6 @@ cmake_minimum_required(VERSION 3.8.2)
 
 set(spm_CONF_FILE ${CMAKE_CURRENT_SOURCE_DIR}/spm.conf)
 
-include($ENV{ZEPHYR_BASE}/../nrf/cmake/boilerplate.cmake)
```

Comment:
IMO there should be a comment in the top of `nrf/cmake/boilerplate.cmake` explaining briefly where to read about the magic that makes this file execute.
Currently you have to read the docs top to bottom in order to figure out how the nrf boilerplate is executed.
A simple pointer to the doc of the feature would suffice,

Reply:
done

Reply:
Look here:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/d175aa30b1717523de10b79c3057c1fb674c9da7/cmake/boilerplate.cmake#L7-L10